packages/apps/kadena-docs/src/components/Layout/components/SideMenu/SideMenu.tsx

```diff
@@ -28,9 +29,21 @@ export const SideMenu: FC<IProps> = ({ closeMenu, menuItems }) => {
     closeMenu,
     menuItems,
   );
+  const router = useRouter();
 
   const activeItem = menuItems.find((item) => item.isMenuOpen);
 
+  const handleKeyPress = (e: KeyboardEvent<HTMLInputElement>): void => {
+    e.preventDefault();
+    const value = e.currentTarget.value;
+    if (e.key === 'Enter') {
+      // eslint-disable-next-line @typescript-eslint/no-floating-promises
+      router.push(`/search?q=${value}`);
+      closeMenu();
+      e.currentTarget.value = '';
```

Comment:
No keycode 13? I didn't know you there's a property with the named key. Does this work in all browsers?